.babelrc

```diff
@@ -2,7 +2,9 @@
   "presets": ["@babel/preset-env"],
   "plugins": [
     "@babel/plugin-proposal-object-rest-spread",
+    "@babel/plugin-proposal-optional-chaining",
     "@babel/plugin-transform-regenerator",
-    "@babel/plugin-transform-runtime"
+    "@babel/plugin-transform-runtime",
+    ["module-resolver", { "root": ["./src"] }]
```

Comment:
This handles aliasing module files so that we can get rid of relative paths in import statements.